File: WoofCare/app/src/main/java/com/example/woofcareapp/api/models/product.kt
package com.example.woofcareapp.api.models

data class Product(
    val id: Int,
    val name: String,
    val description: String,
    val price: Double,
    val location: String,
    val companyName: String,
    val reviewId: Int,//nuevo
    val webUrl: String,//nuevo
    val status: Int,
    val bannerUrls: String //cambiar split , y hacer lista
)

File: BackendApi/entities/Product.py
class Product:
    def __init__(self, id, name, description, price, location, companyName, status, bannerUrls, webUrl):
        self.id = id
        self.name = name
        self.description = description
        self.price = price
        self.location = location
        self.companyName = companyName
        self.status = status
        self.bannerUrls = bannerUrls
        self.webUrl = webUrl

    def to_dict(self):
        return {
            "id": self.id,
            "name": self.name,
            "description": self.description,
            "price": self.price,
            "location": self.location,
            "companyName": self.companyName,
            "status": self.status,
            "bannerUrls": self.bannerUrls,
            "webUrl": self.webUrl
        }
